nieuwe_api/github-api.py

```diff
@@ -0,0 +1,87 @@
+from flask import Flask, json, jsonify, Response, request
+import requests
+
+app = Flask(__name__)
+
+@app.route("/")
+def check_for_user():
+    username = get_username()
+    repo_output_dict = get_repositories(username)
```

Comment:
Kun je hier niet beter met `repos_response` werken in plaats van de output_dict? dus:

```
repos_response = get_repositories(username)
repos_response_list = repos_response.json()
```